ldk/javascript/src/types/oliveHelps/browser.d.ts

```diff
@@ -49,8 +49,9 @@ declare namespace Browser {
   }
 
   interface UIElementArguments {
-    selector: string;
     address: string;
+    listenerType?: string;
```

Comment:
if `listenerType` is limited to certain options, instead of using string directly, we usually create a type `ListenerTypes` and the prop would look like this: 
`listenerType?: ListenType; `

Reply:
good call, made the updates